dunesim/EventGenerator/NuE/NuEScatterGen_module.cc

```diff
@@ -217,8 +234,17 @@ void evgen::NuEScatterGen::reconfigure(fhicl::ParameterSet const & p)
   }
 
   fCustomNuFlavBR = p.get<bool>("CustomNuFlavBR");
+  fFlatESpectrum = p.get<bool>("FlatESpectrum",false);
+  
   if (!fFlatESpectrum || !fCustomNuFlavBR) {
     fEventRateFileName = p.get<std::string>("EventRateFileName");
+    fNpxNueE           = p.get<int>("fNpxNueE",1500);
+    fNpxNumuE          = p.get<int>("fNpxNumuE",1500);
+    fNpxNutauE         = p.get<int>("fNpxNutauE",1500);
+    fNpxNuebarE        = p.get<int>("fNpxNuebarE",1500);
+    fNpxNumubarE       = p.get<int>("fNpxNumubarE",1500);
+    fNpxNutaubarE      = p.get<int>("fNpxNutaubarE",1500);
```

Comment:
The default values could be dropped as they could end up being a point of user error (e.g. if someone typos the name of the variable in their own fcl)